snmp/datadog_checks/snmp/data/default_profiles/_cisco-ipsec-flow-monitor.yaml

```diff
@@ -0,0 +1,83 @@
+metrics:
+  - MIB: CISCO-IPSEC-FLOW-MONITOR-MIB
+    table:
+      name: cikeTunnelTable
+      OID: 1.3.6.1.4.1.9.9.171.1.2.3
+    symbols:
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.15
+        name: cikeTunLifeTime
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.19
+        name: cikeTunInOctets
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.20
+        name: cikeTunInPkts
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.21
+        name: cikeTunInDropPkts
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.27
+        name: cikeTunOutOctets
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.28
+        name: cikeTunOutPkts
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.29
+        name: cikeTunOutDropPkts
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.35
+        name: cikeTunStatus
+    metric_tags:
+      - tag: phase_1_tunnel_index
+        symbol:
+          OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.1
+          name: cikeTunIndex
+      - tag: peer_local_value
+        symbol:
+          OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.3
+          name: cikeTunLocalValue
+      - tag: peer_remote_value
+        symbol:
+          OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.7
+          name: cikeTunRemoteValue
+  - MIB: CISCO-IPSEC-FLOW-MONITOR-MIB
+    table:
+      name: cipSecTunnelTable
+      OID: 1.3.6.1.4.1.9.9.171.1.3.2
+    symbols:
+      - OID: 1.3.6.1.4.1.9.9.171.1.3.2.1.3
```

Comment:
This has the type `TruthValue`:
```
TruthValue ::= TEXTUAL-CONVENTION
    STATUS       current
    DESCRIPTION
            "Represents a boolean value."
    SYNTAX       INTEGER { true(1), false(2) }
```
What do you think of adding this as a tag instead with a mapping to true/false?